noir-projects/aztec-nr/aztec/src/encrypted_logs/log_assembly_strategies/default_aes128/arithmetic_generics_utils.nr

```diff
@@ -0,0 +1,85 @@
+/********************************************************/
```

Comment:
I re-shuffled a lot of code in `encrypted_logs/log_assembly_strategies/default_aes128/` because I needed that to be able to unit test the decryption. Given the state of all the branches it was just impractical to do that in a separate PR.

This code was moved from waht was originally in `encrypted_logs/log_assembly_strategies/default_aes128/note.nr` and does not need to be reviewed.